src/controllers/venueController.ts

```diff
@@ -40,3 +52,5 @@ export const deleteVenue = async (req: Request, res: Response) => {
     return res.status(200).send({ message: 'Successfully deleted venue!' });
   }
 };
```

Comment:
If we're going to do an endpoint that returns one venue and an endpoint that returns multiple venues, we'll have to do separate endpoints for both.

For one venue `('/venues/{venueId}')`
For many venues: `('/venues/?count=10')`

 Endpoints should never have an unpredictable return type (in this case, a single venue or an array of venues). Maybe for the "getMultipleVenues" endpoint you can send up some meta data like the number of venues you want to get and later we add pagination to only load the first 10, and then the next 10, etc.